src/microbe_stage/Microbe.cs

```diff
@@ -662,18 +662,24 @@ public void Kill()
             // Grab random organelle from cell and use that for model
             chunkType.Meshes = new List<Biome.ChunkConfiguration.ChunkScene>();
 
-            var organelleToUse = organelles.Organelles.Random(random).Definition;
-
+            OrganelleDefinition organelleToUse = null;
             var sceneToUse = new Biome.ChunkConfiguration.ChunkScene();
 
-            if (organelleToUse.DisplayScene != string.Empty)
+            foreach (var organelle in organelles.Organelles.OrderBy(_ => random.Next()))
             {
+                organelleToUse = organelle.Definition;
```

Comment:
There's no point in this assignment, right?
This information is not needed after the loop.